jenkinsfiles/new-android.groovy

```diff
@@ -0,0 +1,32 @@
+node('android') {
+
+    stage('Checkout Source') {
```

Comment:
Instead of `  git "https://github.com/feedhenry/some-android-app"`
may use standard checkout without specifying repository in jenkinsfile. 
There are lots of of drawbacks related with this approach, so suggesting previous version:
`checkout scm`

For job examples see our command line client:
See: https://github.com/aerogear/digger-node/pull/1